### PR TITLE
add config working group. 

### PR DESCRIPTION
**Pull request by xiaolanz**

### CODE REVIEW

File: WORKING-GROUPS.md [hunk 1]
```diff
@@ -58,6 +59,25 @@ Document Folder | [Folder](https://drive.google.com/corp/drive/u/0/folders/0BzW5
...
+Community Meeting VC | [Wednesdays at 14:00 PT - Biweekly](https://meet.google.com/istio-config)
+Community Meeting Calendar | [Calendar Invitation](https://calendar.google.com/event?action=TEMPLATE&tmeid=NmtjanE3am1uYmxqcGFldG10MHI2NzBnZnVfMjAxNzExMTVUMjIwMDAwWiB4aWFvbGFuQGdvb2dsZS5jb20&tmsrc=xiaolan%40google.com&scp=ALL)
+Meeting Notes | [Notes](https://docs.google.com/document/d/1P3p7zOpX66hPoZBi_CiC36JW7JmoaLWqE2sgHvdq5tY/edit?ts=5a0b7200)
+Document Folder | [Folder](https://drive.google.com/corp/drive/folders/0B5CC9KT63DznUUQtSU9HTHBnb1E)
+
+&nbsp; | Leads | Company | Profile
+---|---|---|---
+<img width="30px" src="https://avatars0.githubusercontent.com/u/1562325?s=400&v=4"> | Sven Mawson | Google | [smawson](https://github.com/smawson)
+<img width="30px" src="https://avatars3.githubusercontent.com/u/28876302?s=400&v=4"> | Joy Zhang | Google | [xiaolanz](https://github.com/xiaolanz)
+&nbsp; | Jian Guan | Google | [gjwestlife](https://github.com/gjwestlife)
```
Comment (gjwestlife) [inline WORKING-GROUPS.md]: Please include my profile photo.

**Comment by xiaolanz** [comment]
done. PTAL

**Comment by xiaolanz** [comment]
Looks like other WG description key words show a level of details. 
How about:
Config API, config spec, config management, API server, storage, rollout

File: WORKING-GROUPS.md [hunk 2]
```diff
@@ -61,13 +61,13 @@ Document Folder | [Folder](https://drive.google.com/corp/drive/u/0/folders/0BzW5
 
 ## Config
 
-Config API, Config management/rollout
+Config and config management
```
Comment (ldemailly) [inline WORKING-GROUPS.md]: Config and config [...] stutters a bit no ?

Comment (xiaolanz) [reply WORKING-GROUPS.md]: It's fixed. thanks

File: WORKING-GROUPS.md [hunk 3]
```diff
@@ -58,6 +59,25 @@ Document Folder | [Folder](https://drive.google.com/corp/drive/u/0/folders/0BzW5
...
+---|---
+Forum | [istio-config@](https://groups.google.com/forum/#!forum/istio-config)
+Community Meeting VC | [Thursdays at 10:00 PT - Biweekly](https://meet.google.com/istio-config)
+Community Meeting Calendar | [Calendar Invitation](https://calendar.google.com/calendar/event?action=TEMPLATE&tmeid=cWd0MXBrOWM4dm1wZGdqOTV0cDJzbDU2MDhfMjAxNzExMzBUMTgwMDAwWiB4aWFvbGFuQGdvb2dsZS5jb20&tmsrc=xiaolan%40google.com)
+Meeting Notes | [Notes](https://docs.google.com/document/d/1P3p7zOpX66hPoZBi_CiC36JW7JmoaLWqE2sgHvdq5tY/edit?ts=5a0b7200)
+Document Folder | [Folder](https://drive.google.com/corp/drive/folders/0B5CC9KT63DznUUQtSU9HTHBnb1E)
+
+&nbsp; | Leads | Company | Profile
+---|---|---|---
+<img width="30px" src="https://avatars0.githubusercontent.com/u/1562325?s=400&v=4"> | Sven Mawson | Google | [smawson](https://github.com/smawson)
```
Comment (smawson) [inline WORKING-GROUPS.md]: Drop me from the list, I'm not a lead of the WG :)

**Pull request merged**